lib/api.js

```diff
@@ -8,15 +8,43 @@ const api = got.extend({
     "User-Agent":
       "TooGoodToGo/21.9.0 (813) (iPhone/iPhone 7 (GSM); iOS 15.1; Scale/2.00)",
     "Content-Type": "application/json",
-    Accept: "application/json",
+    "Accept": "",
     "Accept-Language": "en-US",
     "Accept-Encoding": "gzip",
   }),
   responseType: "json",
   resolveBodyOnly: true,
 });
 
+function authByEmail() {
+  const credentials = config.get("api.credentials");
+  
+  return api
+    .post("auth/v3/authByEmail", {
+      json: {
+        device_type: "ANDROID",
+        email: credentials.email,
+      },
+    });
+  }
+
+function authPoll(polling_id) {
```

Comment:
It seems like this method is not called anywhere. Surely this PR can't fix our problem, right?